scripts/pack-codegen-for-zig-team.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Was this file added by accident?

Reply:
no. this is an intentional tool to help open bug reports. i wrote this after i accidentally gave mlugg *almost* all of the needed codegen, but i missed two files.